util/stmtsummary/statement_summary.go

```diff
@@ -58,14 +59,23 @@ type stmtSummaryByDigestMap struct {
 	// It's rare to read concurrently, so RWMutex is not needed.
 	sync.Mutex
 	summaryMap *kvcache.SimpleLRUCache
+	// These fields are used for rolling summary.
+	beginTimeForCurInterval int64
+	lastCheckExpireTime     int64
 
-	// enabledWrapper encapsulates variables needed to judge whether statement summary is enabled.
-	enabledWrapper struct {
+	// sysVars encapsulates system variables needed to control statement summary.
+	sysVars struct {
 		sync.RWMutex
 		// enabled indicates whether statement summary is enabled in current server.
 		sessionEnabled string
 		// setInSession indicates whether statement summary has been set in any session.
 		globalEnabled string
+		// XXXRefreshInterval indicates the refresh interval of summaries.
+		// It must be > 0.
+		sessionRefreshInterval string
```

Comment:
If we want > 0, why not use unsigned int?

Reply:
If it's an empty string, then we'll ignore it and use globalRefreshInterval.
This kind of operations was discussed with zhou sir, setting session variable to an empty string indicates unsetting session variable and then use global variable.

Reply:
I see.